pkg/vulnsrc/python/python.go

```diff
@@ -136,3 +137,24 @@ func (vs VulnSrc) Get(pkgName string) ([]Advisory, error) {
 	}
 	return results, nil
 }
+
+func decodeAdvisoryDB(r io.Reader) (AdvisoryDB, error) {
```

Comment:
What if defining `UnmarshalJSON` of `AdvisoryDB`?

Reply:
ok